inc/Plugin.php

```diff
@@ -254,12 +255,15 @@ private function init_common_subscribers() {
 			'amp_subscriber',
 			'rest_cpcss_subscriber',
 			'simple_custom_css',
-			'cloudways',
-			'wpengine',
-			'spinupwp',
 			'pdfembedder',
 		];
 
+		$host_type = HostResolver::get_host_service();
+
+		if ( $host_type ) {
```

Comment:
The data type is a string. Let's be more strict with our data type checks. Use `empty()` or `'' !== $host_type` here to avoid type juggling of a boolean check.

Reply:
Done in https://github.com/wp-media/wp-rocket/pull/2874/commits/9a9a5970182e068a13e9703d1860ef151813e05a